golang/Makefile

```diff
@@ -27,6 +27,7 @@ clean:
 	echo Cleaning build artifacts...
 	go clean
 	rm -rf $(BUILD)
+	rm -rf examples/thrift/gen-go
```

Comment:
Tried to put it in a top level `generated` folder, but that messes up the package naming. Will add to .gitignore.